yarn-project/noir-contracts/src/contracts/docs_example_contract/src/actions.nr

```diff
@@ -36,8 +36,8 @@ pub fn unlock(state_var: PublicState<bool, BOOL_SERIALIZED_LEN>) {
 pub fn get_current_queen(state_var: PublicState<Queen, QUEEN_SERIALIZED_LEN>) -> Queen {
     state_var.read()
 }
-// docs:end:state_vars-PublicStateReadCustom
 
+// docs:end:state_vars-PublicStateReadCustom
```

Comment:
uhhh would prefer reverting this.

Should stay at Line39 rather than at 40, because now it looks like the start

Reply:
I just reverted the changes, installed newest nargo (008a43170aac00b56541f3fa41774614587a0485) and run the formatter again and it indeed no longer did this change.

Great job @f01dab1e!